specification/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -123,6 +146,18 @@ swagger-to-sdk:
   - repo: azure-sdk-for-net
   - repo: azure-sdk-for-python
 ```
+
+### Tag: package-preview-2022-03
+
+These settings apply only when `--tag=package-preview-2022-03` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2022-03'
+input-file:
+  - preview/2022-03-01-preview/common.json
+  - preview/2022-03-01-preview/textanalytics.json
+  - preview/2022-03-01-preview/analyzeconversations.json
+
+
```

Comment:
Seems this block should be removed. You only need a tag filter for tags (versions) that actually exist. This readme.md file probably needs to be cleaned up. I'm betting the other tags like this below are also errant.